ext/web/13_message_port.js

```diff
@@ -216,6 +222,16 @@ class MessagePort extends EventTarget {
     })();
   }
 
+  [refMessagePort](ref) {
```

Comment:
Can you change it to be `ObjectDefineProperty` and make it non-enumerable?

Reply:
Aren't methods of a class non-enumerable by default? What am I missing?

Reply:
Ah, I forgot about that